### PR TITLE
Testing to do List Part 2

### PR DESCRIPTION
**Pull request by kessio**
This pull request project was created by @kessio @iambenkis .
**Project requirements**
- Write unit tests for a JavaScript app.
 - Use jest framework for testing.
 - Test function for editing the task description.
- Test function for updating an item's 'completed' status.
- Test the "Clear all completed" function.
- Use the mock storage object you have created in Part 1 of this project to mock the localStorage updates.
- Mock the HTML elements to test DOM manipulation functions.
- Make sure you group your tests using the describe() method.
-  All the above project requirements has been implemented on this pull Request

### CODE REVIEW

File: src/modules/statusUpdate.js [hunk 1]
```diff
@@ -0,0 +1,21 @@
+// import getTodo from './getTodo.js';
+const selecetedTasks = (todos, isChecked, id) => {
+  // const todos = getTodo().todo;
```
Comment (whiz25) [inline src/modules/statusUpdate.js]: - Please remove this comment. Comments make our code look ugly.

**Review by whiz25 — CHANGES_REQUESTED** [review]
Hi @kessio, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>


## Required Changes ♻️
_Check the comments under the review._
- create a folder called `mocks`. It should contain logic that mocks the functionalities of editing, updating the status of the task and clearing all completed tasks. Use the local storage for this.

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- x
- y
- z

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

### Martin Afani
#### Slack: whiz25

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by mahbubul14 — APPROVED** [review]
# Project Approved :trophy: :tada: 

Hi @Team, 

Your project is complete!  You have done it exceptionally well. it's time to merge it :100: 
Congratulations! 🎉 

### To Highlight :+1: 

:heavy_check_mark: No linter error.
:heavy_check_mark: Descriptive readme file.
:heavy_check_mark: Well implemented suggested changes.

### Optional suggestions :arrow_down_small: 

- [ ] The commit message should be written in the imperative tense. `Ex: "Fix bug" instead of "bug fixed".` [For more details.](https://www.freecodecamp.org/news/level-you-up-to-awesome-commit-messages-a85558cb90e8)

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification. You can also connect with me on [slack](https://microverse-students.slack.com/team/U01J39GL614)

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

**Comment by kessio** [comment]
@mahbubul14 Kindly have a look at our project because you approved it but the dashboard is showing changes required.

**Comment by mahbubul14** [comment]
> @mahbubul14 Kindly have a look at our project because you approved it but the dashboard is showing changes required.

Hi @kessio,
Sorry for your inconvenience. I have already created a ticket to fix the issue. Hope it will be solved asap. If it takes some time, let student success know or text me. I will try to follow up.

**Review by emmanuellekamwa — APPROVED** [review]
**STATUS: APPROVED 🍾✔🎉**
Hello @kessio and @iambenkis 👋
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights

- You documented your work professionally.✔
- Your linters are passing all the checks
- You asked for the changes in a polite and friendly way.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**